### ISSUE TITLE
Unexpected input(s) 'labels'

### ISSUE THREAD

**Issue by yeradis** [google-github-actions/deploy-cloudrun#390]
### TL;DR

When trying to set the 'labels' a warning appears showing: Warning: Unexpected input(s) 'labels', valid inputs are ['image', 'service', 'region', 'env_vars', 'secrets', 'metadata', 'project_id', 'credentials', 'source', 'suffix', 'tag', 'timeout', 'no_traffic', 'revision_traffic', 'tag_traffic', 'flags', 'gcloud_version', 'gcloud_component']

### Expected behavior

Allow to set 'labels' as specified at the README

### Observed behavior

Shows warning: Warning: Unexpected input(s) 'labels', valid inputs are ['image', 'service', 'region', 'env_vars', 'secrets', 'metadata', 'project_id', 'credentials', 'source', 'suffix', 'tag', 'timeout', 'no_traffic', 'revision_traffic', 'tag_traffic', 'flags', 'gcloud_version', 'gcloud_component']

### Action YAML

```yaml
No reason to put all here, the warning message provides all the information. Its used like this:

- name: Deploy to Cloud Run
        id: deploy
        uses: google-github-actions/deploy-cloudrun@v0
        with:
          service: ${{ env.SERVICE }}
          image: ${{ steps.vars.outputs.docker_repository }}:latest
          region: ${{ env.REGION }}
          tag: ${{ env.VERSION }}
          labels: ${{ steps.vars.outputs.labels}}
```
